packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -64,7 +64,6 @@ const labelBaseStyles = ({ theme }: StyleProps) => css`
     box-sizing: border-box;
     height: 18px;
     width: 18px;
-    box-shadow: 0;
```

Comment:
This wasn't doing anything because it's invalid (docs: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)

Removed 🧹